.github/pull_request_template.md

```diff
@@ -0,0 +1,32 @@
+
+**✨ Description:** 
+
+[Please describe the background, purpose, changes made, and how to test this PR]
+
+**🚧 Related Issues:** 
+
+[List the issue numbers related to this PR]
+
+**👨‍💻 Changes Proposed:** 
+
+- [ ] change
+- [ ] ...
+
+
+**🧑‍🤝‍🧑 Who can review?** 
+
+[Please use the '@' symbol to mention any community member who is free to review the PR once the tests have passed. Feel free to tag members or contributors who might be interested in your PR.]
+
+**🛠 TODO**
+
+- [ ] task1
+- [ ] ...
+
+**✅ Checklist:**
```

Comment:
Maybe the following checklist can be added here:
- [ ] Demo/checkpoint has been uploaded (if applicable)